src/NuGet.Clients/NuGet.PackageManagement.UI/Resources/Brushes.cs

```diff
@@ -258,5 +259,20 @@ public static void LoadVsBrushes()
             ListItemBackgroundSelectedColorKey = CommonDocumentColors.ListItemBackgroundSelectedColorKey;
             ListItemTextSelectedColorKey = CommonDocumentColors.ListItemTextSelectedColorKey;
         }
+
+        private static bool IsBackgroundColorFlightEnabled()
+        {
+            var forceFlightEnabled = false;
+            try
+            {
+                forceFlightEnabled = Environment.GetEnvironmentVariable(ExperimentationConstants.EnvironmentVariables.PackageManagerBackgroundColor) == "1";
+            }
+            catch (SecurityException)
+            {
+                // Don't force the flight to be enabled if we are not able to read the environment variable
+            }
+
+            return forceFlightEnabled || ExperimentationService.Default.IsCachedFlightEnabled(ExperimentationConstants.FlightFlags.PackageManagerBackgroundColor);
```

Comment:
I prefer exiting early if `ExperimentationService.Default.IsCachedFlightEnabled(ExperimentationConstants.FlightFlags.PackageManagerBackgroundColor) returns true` due to the following reasons.

- `forceFlightEnabled` can be `true` only for `debugging scenarios`.
-  Calling `Environment.GetEnvironmentVariable` everytime can be avoided.

Reply:
Thanks for the suggestion. Done.